### PR TITLE
feat: polish mobile explore table

### PR DESCRIPTION
**Pull request by lynnshaoyu**
polishes: 
- search bar logo and text used to overlap
- favorites button hide text when 540 px
- reduce columns to just two when 540 px + spacing
- make nav bar full sized instead of squished
- give more space for the token name instead of cutting off to overflow early

540px width: 
Screen Shot 2022-08-10 at 2 06 58 PM

Screen Shot 2022-08-10 at 2 17 09 PM

figma: 
https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-App-Spec-(Phase-0-%26-Phase-1)-%5BWIP%5D?node-id=1708%3A26046

### CODE REVIEW

File: src/components/Explore/TokenTable/SearchBar.tsx [hunk 1]
```diff
@@ -56,6 +56,8 @@ const SearchInput = styled.input<{ expanded: boolean }>`
   }
 `
 
+console.log('test')
```
Comment (kayleegeorge) [inline src/components/Explore/TokenTable/SearchBar.tsx:59]: rm

**Review by kayleegeorge — CHANGES_REQUESTED** [review]

File: src/components/Explore/TokenTable/TokenRow.tsx [hunk 2]
```diff
@@ -247,13 +244,13 @@ const TokenInfoCell = styled(Cell)`
 const TokenName = styled.div`
   overflow: hidden;
   text-overflow: ellipsis;
-  max-width: 160px;
+  max-width: 280px;
```
Comment (kayleegeorge) [inline src/components/Explore/TokenTable/TokenRow.tsx:247]: Screen Shot 2022-08-10 at 3 28 17 PM

Comment (kayleegeorge) [reply src/components/Explore/TokenTable/TokenRow.tsx:247]: @lynnshaoyu i think this change overflows it on bigger screens, maybe make it this max when screen size is SMALL_MEDIA?

Comment (kayleegeorge) [reply src/components/Explore/TokenTable/TokenRow.tsx:247]: Screen Shot 2022-08-10 at 3 29 30 PM

Comment (kayleegeorge) [reply src/components/Explore/TokenTable/TokenRow.tsx:247]: same here

Comment (lynnshaoyu) [reply src/components/Explore/TokenTable/TokenRow.tsx:247]: oof what width was this?

File: src/components/Explore/TokenTable/TokenRow.tsx [hunk 3]
```diff
@@ -175,20 +169,23 @@ const NameCell = styled(Cell)`
...
   gap: 8px;
+
+  @media only screen and (max-width: ${SMALL_MEDIA_BREAKPOINT}) {
+    padding-right: 8px;
+  }
 `
 const PriceCell = styled(DataCell)`
   padding-right: 8px;
 `
 const PercentChangeCell = styled(DataCell)`
```
Comment (kayleegeorge) [inline src/components/Explore/TokenTable/TokenRow.tsx:180]: need the padding-right: 8px here no matter what

File: src/components/Explore/TokenTable/SearchBar.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Explore/TokenTable/SearchBar.tsx]: ty done

File: src/components/Explore/TokenTable/TokenRow.tsx (hunk 3 above)
Comment (lynnshaoyu) [reply src/components/Explore/TokenTable/TokenRow.tsx:176]: sg done

File: src/components/Explore/TokenTable/TokenRow.tsx (hunk 2 above)
Comment (infredible) [reply src/components/Explore/TokenTable/TokenRow.tsx:243]: as an fyi: we should generally make sure our smallest size layout works and doesn't break anything down to 375px (iphone 7/8 size)

Comment (lynnshaoyu) [reply src/components/Explore/TokenTable/TokenRow.tsx]: Screen Shot 2022-08-10 at 3 31 35 PM
this is the newest PR's render

Comment (lynnshaoyu) [reply src/components/Explore/TokenTable/TokenRow.tsx]: this is 375px
Screen Shot 2022-08-10 at 3 49 27 PM

**Review by kayleegeorge — APPROVED** [review]
lgtm! :)

**Pull request merged**